tests/register/register.go

```diff
@@ -107,9 +119,34 @@ func (test Test) UnmountPartitions(t *testing.T, loopDevice string) {
 	util.MustRun(t, "umount", fmt.Sprintf("%sp9", loopDevice))
 }
 
-func (test Test) RunCoreOSInstall(t *testing.T, loopDevice string, opts ...string) {
+func (test Test) GetInstallOptions(t *testing.T, loopDevice string, opts ...string) []string {
 	opts = append(opts, "-d", loopDevice)
 
+	// use local http server if we aren't testing specific flags
+	if test.Version == nil && test.BaseURL == nil && test.Channel == nil && test.Board == nil && !test.UseLocalFile {
+		opts = append(opts, "-b", "http://127.0.0.1:8080/")
+	}
+
+	if test.UseLocalFile && test.LocalImagePath != "" {
```

Comment:
If the test has asked to force a local file, but we don't have a path for one, we should fail rather than silently alter the command line.